Assets/Scripts/Game/UserInterface/SpellIconCollection.cs

```diff
@@ -320,7 +334,7 @@ void LoadPackIcons(SpellIconPack pack, Texture2D atlas, string path)
 
             // Read icons to their own texture (remembering Unity textures are flipped vertically)
             int srcX = 0, srcY = atlas.height - dim;
-            for (int i = 0; i < SpellIconCount; i++)
+            for (int i = 0; i < pack.iconCount; i++)
```

Comment:
Thanks for spotting this, good catch! I'm about to spin up new builds, so I'll quickly make this change and credit you.